flair/file_utils.py

```diff
@@ -99,15 +101,17 @@ def cached_path(url_or_filename: str, cache_dir: Path) -> Path:
         )
 
 
-def unzip_file(file: Path, unzip_to: Path):
+def unzip_file(file: Union[str, Path], unzip_to: Union[str, Path]):
     from zipfile import ZipFile
 
-    with ZipFile(file, "r") as zipObj:
+    with ZipFile(Path(file), "r") as zipObj:
```

Comment:
What if `file is` already a Path?

Reply:
`Path(Path("lkasjdf/lkasjdf"))` executes without error so it shouldn't matter. Indeed probably all the checks for `str` types before conversion could probably be removed but I checked that too late 😅

Reply:
oha didn't know that :D